pySPT/preAnalysis/expNoiseRate_noGUI.py

```diff
@@ -0,0 +1,134 @@
+"""
+@author: Johanna Rahm
+Research group Heilemann
+Institute for Physical and Theoretical Chemistry, Goethe University Frankfurt a.M.
+
+Determine the false positive rate of localizations per cell in %, based on background noise.
```

Comment:
adjust docstring description (add your name under authors, why was this file created, what is the difference to expNoiseRate.py?